### DISCUSSION TITLE
mspider/best-practice/use-egress-and-authorized-policy

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2928] [Announcements]
# mspider/best-practice/use-egress-and-authorized-policy

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/best-practice/use-egress-and-authorized-policy.html

**Comment by hanxiaop** [comment]
网格启用仅出口流量修改为网格出站流量策略启用为仅注册服务
